### DISCUSSION TITLE
Start page default links

### DISCUSSION THREAD

**Discussion by galeksandrp** [bromite/bromite#2187] [Builds]
Could you please point me to place in code where are start page default links (chromium.org, bromite.org, eff.org, libera.chat, chrome://flags) are defined?

**Comment by csagan5** [comment]
I assume you want to change them and/or do some rebranding; search for "popular sites" in the Chromium codebase and you will find them.

**Reply by galeksandrp** [reply]
Yes, they always been in [default_popular_sites.json](https://source.chromium.org/chromium/chromium/src/+/main:components/ntp_tiles/resources/default_popular_sites.json), all that's left is to find which `Bromite` patch modifies them.

Cannot find any with `grep`, I'm missing something?

**Reply by csagan5** [reply]
The patch that changes them (together with the changes for branding/logo) is not released, on purpose. No plans to change that either.
